app/javascript/mastodon/locales/en.json

```diff
@@ -109,7 +109,7 @@
   "confirmations.block.confirm": "Block",
   "confirmations.block.message": "Are you sure you want to block {name}?",
   "confirmations.delete.confirm": "Delete",
-  "confirmations.delete.message": "Are you sure you want to delete this toot?",
+  "confirmations.delete.message": "Are you sure you want to delete this status?",
```

Comment:
@Gargron There seem to be many places where "toot" has been changed back to "status"